packages/wasm/src/services/credential/bound-check.test.ts

```diff
@@ -261,32 +261,4 @@ describe('Bound check', () => {
       'Unsupported format undefined and type string for enforce bounds',
     );
   });
-
```

Comment:
?

Reply:
this commit is from this PR https://github.com/docknetwork/react-native-sdk/pull/155. We are dropping the dateTimeToTimestamp function as the crypto-wasm-ts will implement it. 

let me rebase this PR

Reply:
it's done, the branch is rebased